app-editors/levee/levee-4.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Really tiny vi clone, for things like rescue disks"
+HOMEPAGE="https://www.pell.portland.or.us/~orc/Code/levee/"
+SRC_URI="https://www.pell.portland.or.us/~orc/Code/levee/${P}.tar.bz2"
+
+LICENSE="levee"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
+
+RDEPEND="
+	!app-text/lv
+	sys-libs/ncurses:0=
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-fix-build-for-clang16.patch
+	"${FILESDIR}"/${P}-tgetent-ncurses-gentoo.patch
+)
+
+src_configure() {
+	export AC_CPP_PROG="$(tc-getCPP)"
+	export AC_PATH="${PATH}"
+	export AC_LIBDIR="$($(tc-getPKG_CONFIG) --libs curses ncurses termcap)"
+
+	./configure.sh \
+		--prefix="${PREFIX}"/usr --size=256000 || die "configure failed"
+}
+
+src_compile() {
+	emake \
+		CFLAGS="${CFLAGS} -Wall -Wextra ${LDFLAGS}" \
```

Comment:
I don't think we really need -W* here tbh

Reply:
yeah, I got this from the old ebuild. Upstream seems to set much store on the fact that his stuff builds with that. 

I will remove it.